pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -772,40 +839,59 @@ func (p *MasqueradePodInterface) createNatRulesUsingIptables() error {
 	return nil
 }
 
-func (p *MasqueradePodInterface) createNatRulesUsingNftables() error {
-	err := Handler.NftablesLoad("ipv4-nat")
-	if err != nil {
-		return err
+func getGatewayByProtocol(p *MasqueradePodInterface, proto iptables.Protocol) string {
+	if proto == iptables.ProtocolIPv4 {
+		return p.gatewayAddr.IP.String()
+	} else {
+		return p.gatewayIpv6Addr.IP.String()
 	}
+}
+
+func getVifIpByProtocol(p *MasqueradePodInterface, proto iptables.Protocol) string {
```

Comment:
Also here, why not a method?

Reply:
Done, I hope I understood your intention correctly.